corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -541,6 +541,24 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
               </div>
             </div>
           {% endif %}
+          {% if request|toggle_enabled:"RESTRICT_LOGIN_AS" %}
+            <div class="form-group">
+            <label class="col-sm-4 control-label">
+              {% trans "Login-As All Users" %}
```

Comment:
Is the way that this works that the login as all users flag is the default assumption, and the "feature" is allowing unchecking the box? 

That does make sense, I'm just confirming that I get it, since it's a bit counter-intuitive.

Reply:
Yeah, that is correct. All of our permissions currently "positive", in that checking the box gives you more permission, so I did not want to break that convention.